src/js/components/NavTabsComponent.jsx

```diff
@@ -1,21 +1,38 @@
 var classNames = require("classnames");
 var React = require("react/addons");
 
+var DeploymentEvents = require("../events/DeploymentEvents");
+var DeploymentStore = require("../stores/DeploymentStore");
+var DeploymentActions = require("../actions/DeploymentActions");
+
 var NavTabsComponent = React.createClass({
   displayName: "NavTabsComponent",
 
   propTypes: {
     activeTabId: React.PropTypes.string.isRequired,
+    activeDeployments: React.PropTypes.number,
```

Comment:
@aldipower I introduced this to allow testing (see below), but I don't like it at all. Do you think it makes sense to rewrite the tests so that the activeDeployments value is take from the DeploymentStore? I gave it a shot but ran into some `promise` issues.